src/main/java/com/example/petbuddybackend/service/chat/ChatService.java

```diff
@@ -113,6 +149,8 @@ private ChatMessageDTO createChatRoomWithMessageForClientSender(
         ChatRoom chatRoom = createChatRoom(clientSender, caretakerReceiver);
         ChatMessage chatMessage = persistMessage(chatRoom, clientSender.getAccountData(), message.getContent());
 
+        chatMessageRepository.updateUnseenMessagesOfClient(chatRoom.getId(), clientSenderEmail);
```

Comment:
Tutaj tworzymy wiadomosc od jakiegos klienta do opiekuna. Więc czemu oznaczamy wszystkie wiadomosci, w ktorych wysylajacym byl klient jako przeczytane?

Reply:
okej, naznaczamy tak wszystkie, w ktorych opiekun byl wysylajacym. Czyli ze jak cos wyslemy to z automatu mamy przeczytane